src/sass/_header.scss

```diff
@@ -34,18 +34,14 @@
         top: -18px;
         width: 20px;
         height: 12px;
-        background-image: url('./images/Burger.svg');
+        background-image: url(./images/Burger.svg);
```

Comment:
musza byc 2 kropki na poczatku, to nie bedzie dzialac
trzeba pisac sciezke w odniesieniu do main.scss